### DISCUSSION TITLE
[node] needs Fetch API types

### DISCUSSION THREAD

**Discussion by saschanaz** [DefinitelyTyped/DefinitelyTyped#63643] [Issues with a @types package]
Fetch API is available as of version 18 but it's not in the type package.

* https://nodejs.org/api/globals.html#fetch
* https://nodejs.org/api/globals.html#class-formdata
* etc.

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/60924
